### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by maddhruv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "install-types",
-  "version": "2.0.1",
+  "version": "2.1.0",
```
Comment (rheaditi) [inline package.json:3]: why minor?

Comment (maddhruv) [reply package.json:3]: we can do `patch` as well, as there are just packages/dependencies updates

Comment (rheaditi) [reply package.json:3]: let's do a patch 👍🏽

Comment (maddhruv) [reply package.json]: Done 🤗

**Review by rheaditi — APPROVED** [review]

**Pull request merged**